src/Components/MediaCard.js

```diff
@@ -0,0 +1,96 @@
+import ReactPlayer from 'react-player'
+import React, { useState, useEffect } from 'react'
+import Media from './Media'
+import { BrowserRouter as Router } from 'react-router-dom'
+import axios from 'axios'
+
+
+
+
+export default function MediaCard({ data }) {
+    const [title, setTitle] = useState('')
+    const [mediaType, setMediaType] = useState('')
+    const [media, setMedia] = useState('')
+    const [ShowMedia, setShowMedia] = useState(false)
+    const [like, setLike] = useState(false)
+
+    const info = () => {
```

Comment:
this looks like a copy and paste from the Media component. Find a way to pass this as a props or something. You should not be repeating yourself